nss-cohort-36/bangazon-api-illin-illusionists-api
# Jwo orders create del methods

# Description

Finalized the POST and DEL methods for orders 

Fixes # N/A

## Type of change


## Testing Instructions

* Ensure all of your local changes have been commited prior running the commands in the next step

* After being assigned as a reviewer for the PR, click the "check out locally" button, then copy and paste the commands into your terminal.

1. Open postman and paste the following URL into your request bar `http://localhost:8000/orders`.
2. Perform a `GET` request to the root and verify a list of order objects is returned.
3. Perform a `GET` request to `http://localhost:8000/orders/1` and verify single  order object is 
    returned
4. Perform a `POST` request with the following body to `http://localhost:8000/orders` and verify it has been added to the data on the API testing site at `http://127.0.0.1:8000/orders?format=api`. 
    ```json
    {
        "payment_type": 1
    }
    ```
5. Perform a `DEL` request to `http://localhost:8000/orders/[YOUR ID HERE]`, then verify it was deleted via the API testing site at `http://127.0.0.1:8000/orders?format=api`.

## Checklist:
